src/background/datastore.js

```diff
@@ -133,13 +134,16 @@ class DataStore {
       // assume formSubmitURL === hostname
       info.formSubmitURL = info.hostname;
     }
-    // TODO: call logins API
+
     let added = {
       ...info,
       guid: UUID(),
+      timesUsed: 0,
+      timeLastUsed: Date.now(),
       timeCreated: Date.now(),
       timePasswordChanged: Date.now(),
     };
+    await browser.experiments.logins.add(added);
```

Comment:
This and other logins-modifying API methods do throw if something goes wrong, so I'd wrap them in try/catch blocks and just log the errors for now. I think we can handle the errors in the relevant issues (looks like #18, #30, #34 might all be relevant).

Reply:
Hmm, I can just silently log them, though datastore.js *does* throw errors for other validation problems and such. Kind of figured it was useful to let these throw too, though it doesn't look like we do anything useful with them yet. (e.g. report an error over a message port, etc)

Reply:
> datastore.js does throw errors for other validation problems and such

Ah, ok. Probably fine as is, then.